search-commons/build.gradle

```diff
@@ -46,7 +46,7 @@ test {
     environment "API_HOST", "localhost"
     environment("SEARCH_INFRASTRUCTURE_API_HOST", "localhost")
     environment("SEARCH_INFRASTRUCTURE_AUTH_URI", "localhost")
-    environment("SEARCH_INFRASTRUCTURE_API_URI", "localhost")
+    environment("SEARCH_INFRASTRUCTURE_API_URI", "https://localhost")
```

Comment:
HttpRequest getHttpRequest(URI requestUri, String mediaType)  validates the URI, and localhost isn't accepted